composer.json

```diff
@@ -7,9 +7,9 @@
     "require": {
         "php": "~7.4",
         "sylius/sylius": "^1.8",
-        "gedmo/doctrine-extensions": "^2.4.12",
+        "gedmo/doctrine-extensions": "^2.4.12 || ^3.0",
         "monsieurbiz/sylius-rich-editor-plugin": "^2.0@RC",
-        "phpstan/phpstan-webmozart-assert": "^0.12.7"
+        "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev"
```

Comment:
Why?

Reply:
Because it's needed to be compatible with Sylius 1.9 : https://github.com/Sylius/Sylius/blob/master/UPGRADE-1.9.md

Reply:
Moved to require-dev